projects/robots/boston_dynamics/spot/protos/Spot.proto

```diff
@@ -0,0 +1,1459 @@
+                ]

```

Comment:
Still empty.

Reply:
I don't know why you don't see anything because it is not empty. Maybe it is a Github bug....

| ![Capture](https://user-images.githubusercontent.com/36080014/78753339-e6e31e00-7975-11ea-818d-3c8a5903913a.JPG)  | ![Capture1](https://user-images.githubusercontent.com/36080014/78753361-f2364980-7975-11ea-8621-8d6035d95cf4.JPG) |
| --- | --- |
| ![Capture2](https://user-images.githubusercontent.com/36080014/78753385-fd897500-7975-11ea-9595-cc806700859a.JPG) | ![Capture3](https://user-images.githubusercontent.com/36080014/78753411-0712dd00-7976-11ea-91c4-f2cacff08f42.JPG) |

Reply:
Indeed, it seems the diff on the PR is completely wrong, but if I go directly on your branch I can see the up to date version without this issue.
That's not going to help for the review.